config/setup_consul_no_secrets.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
```

Comment:
Or wait, we still have `settings.sh`?  Is that because we have the option of using consul OR environ vars in development?  (In deployment we'll just use consul, right?)